unit_of_time/__init__.py

```
@@ -136,6 +136,17 @@ def get_next(cls, dt):
     def to_str(cls, dt):
         return dt.strftime(cls.formatter)
 
+    def get_index_for_date(cls, dt):
+        return None
+
+    def get_date_from_index(cls, dt):
```

Ruff (0.13.3)

Unused method argument: `dt`

(ARG002)